vowpalwabbit/comp_io.cc

```diff
@@ -1,76 +1,148 @@
-#include "zlib.h"
 #include "comp_io.h"
```

Comment:
Have you computationally measured the impact of this alternative access to zlib?